Automattic/jetpack
# Admin Page: use a dynamic version in cache buster on dev env.

#### Changes proposed in this Pull Request:

* When `WP_DEBUG` is active, we could ensure the admin bundle loads a more recent (i.e. last time the bundle was changed) instead of using the more stable `JETPACK__VERSION`.

#### Jetpack product discussion

* ﻿Related discussion: p1646895372007269-slack-CDLH4C1UZ

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On your Docker instance, go to Jetpack > Dashboard
* Check the network tab
* You should see the version number be a unique hash for `_inc/build/admin.js`, instead of the plugin's version number.
